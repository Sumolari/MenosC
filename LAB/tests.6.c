{
	int numero[7 ];
	numero = 20;

	bool boleano;
	boleano = false;
	bool boleano[57];

	if ( 5 ) {} else {}

	if ( 5 ) numero = 3; else numero = 5;

	if ( numero >= 4 )
		numero = 3;
	else
		numero = 5 + 5;

	numero = 30;

	if ( numero >= 4 )
		numero = 3;
	else
		numero = 5 + 5;

	if (numero>=4)
		numero = 3;
	else
		numero = 5+5;

	if(numero>=4)numero=3;
	else numero=5+5;

	if(numero>=4)numero=3;else numero=5+5;
}
