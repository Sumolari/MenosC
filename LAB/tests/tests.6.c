{
	int numero[7 ];
	numero = 20;

	bool boleano;
	boleano = false;
	bool boleano[57];

	if ( 5 ) {} else {}

	if ( 5 ) numero = 3; else numero = 5;

	if ( numero >= 4 )
		numero = 3;
	else
		numero = 5 + 5;

	numero = 30;

	if ( numero[3] >= 4 )
		numero[6] = 3;
	else
		numero[5] = 5 + 5;

	if (numero[2]>=4)
		numero = 3;
	else
		numero = 5+5;

	if(numero>=4)numero=3;
	else numero[7]=5+5;

	if(numero>=4)numero=3;else numero=5+5;
}
