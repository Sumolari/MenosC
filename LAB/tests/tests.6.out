Linea 3: Tipo incompatible
Linea 7: Redeclaracion de variable
Linea 9: Tipo incompatible
Linea 11: Tipo incompatible
Linea 11: Tipo incompatible
Linea 11: Tipo incompatible
Linea 13: Tipo incompatible
Linea 14: Tipo incompatible
Linea 16: Tipo incompatible
Linea 18: Tipo incompatible
Linea 26: Tipo incompatible
Linea 28: Tipo incompatible
Linea 30: Tipo incompatible
Linea 34: Tipo incompatible
Linea 34: Tipo incompatible
Linea 34: Tipo incompatible

Numero de errores:      16
